### DISCUSSION TITLE
Adding Constraints for Validating

### DISCUSSION THREAD

**Discussion by ErikRicci** [lcobucci/jwt#646] [Q&A] [answered]
Hey, it might be a really dumb question, but how does one add constraint before validating a token? 🗡️ thx

Also, eu sou do Brasil xD

**Comment by ErikRicci** [comment, marked as answer]
Nevermind, I've managed to do it 😲

If someone in the future stuble upon this, here you go 🙏
![image](https://user-images.githubusercontent.com/8097564/106215504-e2f7e500-61af-11eb-886a-a95e16a1fe6d.png)

I'm pretty sure it has a easier way of doing it, but there's my try. B)

**Reply by lcobucci** [reply]
Hey @ErikRicci!

>  it might be a really dumb question

There are no dumb questions! Some details might be too obvious for me and not obvious at all for others :smile: 

> Also, eu sou do Brasil xD

Aeeee
FIco feliz em saber que ainda posso ajudar o povo no Brasil tb  :smile: 

> If someone in the future stumbles upon this, here you go pray

That's indeed the general idea. The config object isn't **really** mandatory but its purpose is mainly to allow for people to define how the library will behave only once (storing the config object on the DI container even) and then using the object to get the defined rules.

Would like to send us a PR to improve the docs?
